nix/testresults.nix

```diff
@@ -0,0 +1,19 @@
+{ curl, reflexPlatform, testdriver, testserver, phantomjs }:
+
+reflexPlatform.nixpkgs.pkgs.runCommand "runWebdriveTest.sh" {} ''
+  echo About to phantom
+  ${phantomjs}/bin/phantomjs --webdriver=127.0.0.1:4444 &
+  sleep 3
+  echo About to Server
+  STATIC_DIR=${testserver}/static ${testserver}/back -q --no-access-log --no-error-log -p 8000 &
+  sleep 3
+  ${curl}/bin/curl localhost:8000
+  ${curl}/bin/curl localhost:8000/runmain.js
+  echo About to testdrive
+  ${testdriver}/bin/spec
```

Comment:
From the name `testresults` I would expect `${testdriver}/bin/spec > $out 2>&1` or something similar

Reply:
Great. The actual test results are much more informative than `Done`! :)

```
[greghale@p51:~/code/servant-reflex]$ cat result

Servant.Reflex
  servant-reflex tests
    test page using chrome
      opens the page
      has a div
      has demo-group div
      doesn't have demo-group-nonexistent div
      can get an int or unit from the API
      handle query params
      handles queryflag
      tags dyn promptly

Finished in 3.1099 seconds
8 examples, 0 failures
```